### DISCUSSION TITLE
Stack component

### DISCUSSION THREAD

**Discussion by JordanMarr** [sydsutton/FS.FluentUI#3] [General]
I'm currently perusing the FluentUI 9 docs and trying to decide if it's worth the ugprade from 8 to 9.
It looks like the `Stack` component went away, although I thought I saw it somewhere in your codebase.
Was it removed in 9? I've made pretty extensive use of that control in my app, so it's kind of deterring me from moving forward if true.

**Comment by sydsutton** [comment]
Hey @JordanMarr. Yes, it was removed in v9, but I created a separate package named `FS.FluentUI.V8toV9` that has Stack and StackItem. The only thing that I couldn't figure out with `Fui.stack` was how to style the stack with `stack.style`. I've been using `stack.className` for styling instead, which sometimes is not ideal. Other than that prop, it should be complete.

**Comment by JordanMarr** [comment]
I wonder if removing it was an attempt to nudge users towards using CSS instead.

**Reply by sydsutton** [reply]
Yeah, maybe. If it was, then it's working. But yes, FS.FluentUI.V8toV9 gives you the ability to continue using `Stack` if you did want to migrate to V9.
